rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/IBaseLanguageServerExtensions.java

```diff
@@ -59,6 +55,11 @@ default CompletableFuture<Two<String, URI[]>[]> supplyPathConfig(PathConfigParam
     @JsonNotification("rascal/vfs/register")
     void registerVFS(VFSRegister registration);
 
+    @JsonNotification("rascal/supplyRemoteIDEServicesConfiguration")
+    default CompletableFuture<IDEServicesConfiguration> supplyRemoteIDEServicesConfiguration() {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
why did this move?

Reply:
No particular reason; that's an artefact of how this code evolved. I'll move it up